cinema/serializers.py

```diff
@@ -45,6 +56,24 @@ class Meta:
         fields = ("id", "title", "description", "duration", "genres", "actors")
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ["id", "row", "seat", "movie_session"]
+        validators = [
+            UniqueTogetherValidator(
+                queryset=Order.objects.all(),
+                fields=["movie_session", "row", "seat"]
+            )
+        ]
```

Comment:
No need to set validator here, since you have one in models.
Try removing validators and write the following in the python console:
from cinema.serializers import TicketSerializer
serializer = TickerSerializer
print(repr(serializer)